docs/changelog.md

```diff
@@ -7,6 +7,7 @@
   implementing this interface.
 - Add back scan data and back scan settings getter and setter methods
   to `qudi.interface.scanning_probe_interface`.
+- Moved ConfigOption ScannerGui.optimizer_plot_dimensions to qudi.logic.scanning_optimize_logic and optimizer sequence creation is solely handled by logic
```

Comment:
need to mention the new name of the ConfigOption "optimizer_sequence_dimensions"